src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -337,7 +325,6 @@ public StorageClientFactory StorageClientFactory
         /// </summary>
         public void UseDevelopmentSettings()
         {
-            Tracing.ConsoleLevel = TraceLevel.Verbose;
```

Comment:
There isn't an ILogger analog to this? Configure the logger for maximal debug output?

Reply:
There may be something we can do -- like change the default logger level up to Debug.

Reply:
This is a little tougher to do now since the Filter is created by the user and we don't have direct access to it -- I'll need to refactor. I've filed https://github.com/Azure/azure-webjobs-sdk/issues/1471.